### ISSUE TITLE
ActionButton toggles does not have sufficient color contrast between pressed and not pressed

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#755]
### Expected Behaviour
There should be adequate contrast between the pressed and not

### Actual Behaviour
The difference between pressed and not pressed does not meet the required 3:1 luminosity contrast ratio requirements

### Reproduce Scenario (including but not limited to)
https://opensource.adobe.com/spectrum-web-components/components/action-button

#### Steps to Reproduce

- run page get color of untoggled button
- get color of pressed button
- note there is not a 3:1 ratio between them
- (can use https://webaim.org/resources/contrastchecker/ to verify)

#### Platform and Version

tested lightest theme

#### Sample Code that illustrates the problem
https://opensource.adobe.com/spectrum-web-components/components/action-button
![Screen Shot 2020-07-10 at 4 28 50 PM](https://user-images.githubusercontent.com/1724479/87210549-c60acb80-c2ca-11ea-9090-e37c3ede47b4.png)

**Comment by Westbrook** [comment]
We're comparing the background color in these two cases, correct? This comes out as https://webaim.org/resources/contrastchecker/?fcolor=FAFAFA&bcolor=EAEAEA here and at the Spectrum CSS level. I'll raise this issue there for review at the specification level.

**Comment by Westbrook** [comment]
@jnurthen would you happen to have a docs link to the desire for 3:1 luminosity here? We're looking to make sure we have all the right context to lock this down in the DNA pipeline.

**Comment by jnurthen** [comment]
This is to meet WCAG 1.4.1 - Use of Color
https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html includes information on why 3:1 is enough
